frontend/test/metabase/scenarios/admin/datamodel/metadata.cy.spec.js

```diff
@@ -0,0 +1,36 @@
+import {
+  restore,
+  signInAsAdmin,
+  openOrdersTable,
+  popover,
+} from "__support__/cypress";
+
+describe("scenarios > admin > datamodel > metadata", () => {
+  before(restore);
+  beforeEach(signInAsAdmin);
+
+  it("should correctly show remapped column value", () => {
+    // go directly to Data Model page for Sample Dataset
+    cy.visit("/admin/datamodel/database/1");
+    // edit "Product ID" columnn in "Orders" table
+    cy.findByText("Orders").click();
+    cy.findByDisplayValue("Product ID")
+      .parent()
+      .find(".Icon-gear")
+      .click();
+
+    // remap its original value to use foreign key
+    cy.findByText("Use original value").click();
+    cy.findByText("Use foreign key").click();
```

Comment:
This is still referring to foreign key remapping rather than number-to-string remapping. I think that's fine though, I'm pumped to get this merged and remove e2e. There's plenty of time for tweaking cypress tests down the line.